Add build step (#1)
louisescher/astro-transition-event-polyfill: package/README.md

## Usage / How it works

As long as the integration is used, the `astro:page-load` event will be fired on all pages that don't have view transitions enabled.

> 
> 

Reviewer: ```suggestion

```

## Pre-configured Build/Dev commands

**Commands**

> - **`pnpm dev`** - Runs the playground, and starts the `esbuild` in DEV mode.

Reviewer: ```suggestion
- **`pnpm dev`** - Runs the playground and starts `esbuild` in development mode.
```

  - Runs: `pnpm scripts dev 'src/**/*.ts'`
- **`pnpm build`** - Runs `esbuild`.
  - Runs: `pnpm scripts build 'src/**/*.ts'`

> ## The `scripts/` folder

Reviewer: ```suggestion
## The `scripts/` Folder
```

> Scripts is a CLI project that wraps `esbuild` with nice console output. As well as provides two modes, `build` and `dev`.  Dev mode creates a `esbuild` context and watches for file changes. this creates a scripts command usable when scripts is a workspace dependency.

Reviewer: ```suggestion
Scripts is a CLI project that wraps `esbuild` with a nice console output as well as providing two modes, `build` and `dev`.  Dev mode creates an `esbuild` context and watches for file changes. This creates a scripts command usable if scripts is a workspace dependency.
```
